modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -961,9 +961,15 @@ namespace
             {
                 std::set<int> in_hs, out_ws, out_hs;
 
+                //DELETE ME
+                bool has_garray = false;
+
                 for (const auto& in : node->inNodes())
                 {
                     const auto& d = g.metadata(in).get<Data>();
+                    if(d.shape == cv::GShape::GARRAY){
+                        has_garray = true;
+                    }
```

Comment:
I believe you don't need to change anything here

Reply:
I mean, this new code is not needed

Reply:
Removed